src/ios/VLCPlugin.m

```diff
@@ -630,6 +656,24 @@ - (NSString*) vlc_convertAudioStateToString:(int) state{
 
 }
 
+- (void)vlc_resume {
+    if (self.activeMediaType == VLCPluginMediaTypeOnDemand) {
+        if (self.activePK) {
+            NSString *js = [NSString stringWithFormat:@"window.localStorage.getItem('audio-status-%@')", self.activePK];
```

Comment:
I don't know precisely what @bkammin had in mind, but we have an AudioCallback method in wnyc-app-www's js/NYPRNativeFeatures which seems appropriate.  e.g. that would be called with result.type as 'audioneedstatus' or something and the js would callback with the getItem('audio-status' + result.audio) or something.